### ISSUE TITLE
Add options send notifications for 'save attribute' & 'delete attribute'

### ISSUE THREAD

**Issue by YuriyLytvynchuk** [thingsboard/thingsboard#7667]
Save atrribute node:
 - add option "sendAttributesUpdatedNotification"
![image](https://user-images.githubusercontent.com/102864764/203326365-5a527951-bb87-4733-a20c-9d0ab5d9ac1a.png)

Delete atrribute node:
- add option "sendAttributesDeletedNotification"
![image](https://user-images.githubusercontent.com/102864764/203326617-cc55265b-a470-4f0c-b60f-0e61537b2a51.png)

**Comment by ShvaykaD** [comment]
Hi @ashvayka, final logic for the "Save Attributes" node was updated. Please check the node details below:

Saves entity attributes based on configurable scope parameter. Expects messages with the 'POST_ATTRIBUTES_REQUEST' message type. Rule node allows user to enable/disable sending attributes updated notifications for <b>SHARED_SCOPE</b> and <b>SHARED_SCOPE</b> attributes updates. If upsert(update/insert) operation is completed successfully rule node will send the incoming message via <b>Success</b> chain, otherwise, <b>Failure</b> chain is used. Additionally if checkbox <b>Send attributes updated notification</b> is set to true, rule node will send the "Attributes Updated" event to the root chain of the message originator.

In other words, the rule node allows sending attributes updated notifications to the rule engine only if the scope is set to SHARED_SCOPE or SERVER_SCOPE. The same logic we have in the save attributes API. 

No additional changes were added for the delete attributes node cause in the delete attributes API, we send notifications to the rule engine for any attribute scopes.
